### PR TITLE
unit test for task 3

### PR DESCRIPTION
**Pull request by vkhomyn95**

### CODE REVIEW

File: master_writer.ino [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+void loop() {
+ Wire.beginTransmission(8);
+ unsigned long send = 1; /* begin with device address 8 */
+ Wire.write(send);  /* sends hello string */
+ Wire.endTransmission();    /* stop transmitting */
+
+ Wire.requestFrom(8, 1); /* request & read data of size 12 from slave */
+ while(Wire.available()){
```
Comment (andrii-s-shpitser-lpnu-ua) [inline master_writer.ino:16]: how your program know what to do: run test or normal work?

File: master_writer.ino [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    unsigned long response = Wire.read();/* read data received from slave */
```
Comment (andrii-s-shpitser-lpnu-ua) [inline master_writer.ino:17]: muster should change messege and should not comunicate with terminal. And I see that your muster and slave do not comunicating between themselfs - they just comunicating with terminal. Or Am I wrong?

File: slave_receiver.ino [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ Serial.println("I am I2C Slave");
+}
+
+void loop() {
+ delay(100);
+}
+
+// function that executes whenever data is received from master
+void receiveEvent(int howMany) {
+ while (0 < Wire.available()) {
```
Comment (andrii-s-shpitser-lpnu-ua) [inline slave_receiver.ino:17]: it does not counicating with muster...

File: master_writer.ino (hunk 1 above)
Comment (vkhomyn95) [reply master_writer.ino:16]: Can i configure by using variable at the top of my code?

Comment (andrii-s-shpitser-lpnu-ua) [reply master_writer.ino:16]: in that case you will need reprogram device each time you want to change mode. Better solution is to use some button or read comand from terminal

File: master_writer.ino (hunk 2 above)
Comment (vkhomyn95) [reply master_writer.ino:17]: terminal is jus for output info

Comment (andrii-s-shpitser-lpnu-ua) [reply master_writer.ino:17]: only slave should comunicate with terminal

**Pull request closed without merging**